### DISCUSSION TITLE
Welcome to vue-axios Discussions!

### DISCUSSION THREAD

**Discussion by imcvampire** [imcvampire/vue-axios#79] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by laughm** [comment]
vue 2.6.14 install vue-axios 3.5.0 error
vue-cli's version is 5.0.8
